### PR TITLE
GPG2 Fixes

### PR DESCRIPTION
**Pull request by itay-grudev**
* Removed --no-tty as it disables the card admin PIN pinentry in `initrd/bin/kexec-sign-config`
* Added the `CONFIG_GPG2` flag in the Librem boards
* Disabled `libsecret` support in the pinentry module

### CODE REVIEW

**Comment by jandryuk** [comment]
@tlaurion specifically had to add --no-tty here: https://github.com/osresearch/heads/commit/cc89d6832d6bb0a4607cf2e331341e8bf9c9aa3f

**Comment by itay-grudev** [comment]
I think the original issue no longer exists after setting the `GPG_TTY=$(tty)`. And now it fails because it can't prompt for PIN.

File: initrd/bin/kexec-sign-config [hunk 1]
```diff
@@ -49,7 +49,7 @@ if [ -z "$param_files" ]; then
 fi
 
 for tries in 1 2 3; do
-	if sha256sum $param_files | gpg --no-tty\
+	if sha256sum $param_files | gpg\
```
Comment (tlaurion) [inline initrd/bin/kexec-sign-config]: Aesthetic space required:
`if sha256sum $param_files | gpg \`

Comment (itay-grudev) [reply initrd/bin/kexec-sign-config]: Done.

**Comment by itay-grudev** [comment]
Ping!

**Pull request merged**